### PR TITLE
Update Twitter to X

### PR DESCRIPTION
**Pull request by validcube**
X, possibly the most controversial changes in Lawnchair 12.1

## Description

> [!NOTE]
> X (Twitter)'s SVG file is provided by Font Awesome (not the holder of this trademark)

This changes the URL, text, and icon to X.

### Testing

📱Lawnchair was verified to be working fine on Samsung M23 5G, One UI 5.1 (33) @ August Security Patch.

## Type of change

✅ General change (non-breaking change that doesn't fit the below categories like copyediting)
:x: Bug fix (non-breaking change which fixes an issue)
:x: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: lawnchair/res/drawable/ic_x_twitter.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<vector android:height="24dp" android:viewportHeight="512"
+    android:viewportWidth="512" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#FF000000" android:pathData="M389.2,48h70.6L305.6,224.2 487,464H345L233.7,318.6 106.5,464H35.8L200.7,275.5 26.8,48H172.4L272.9,180.9 389.2,48zM364.4,421.8h39.1L151.1,88h-42L364.4,421.8z"/>
```
Comment (Goooler) [inline lawnchair/res/drawable/ic_x_twitter.xml]: Need reformating.

**Comment by KTibow** [comment]
Why when `x.com` still redirects to `twitter.com`

**Comment by validcube** [comment]
> Why when `x.com` still redirects to `twitter.com`

Ah well!

Twitter has changed its logo and its name to a literal X. So, for me, it would make sense to also change the URL to **x.com**.

File: lawnchair/res/drawable/ic_x_twitter.xml (hunk 1 above)
Comment (validcube) [reply lawnchair/res/drawable/ic_x_twitter.xml]: > Need reformating.

Done!

**Pull request merged**